boards/adi/max32670evkit/max32670evkit.yaml

```diff
@@ -11,5 +11,6 @@ supported:
   - gpio
   - serial
   - trng
+  - i2c
```

Comment:
This needs to be added to the max32655 boards too

Reply:
Done.